tests/test_unify.py

```diff
@@ -234,14 +233,14 @@ def test_engine_parsers_msgf_unified_frame():
         },
     )
     df = u.get_dataframe()
-    assert isinstance(df, UnifiedDataFrame)
-    assert len(df) == 91
+    assert isinstance(df, pd.DataFrame)
+    assert len(df) == 92
     assert (
-        df.df.iloc[0]["Protein ID"]
+        df.iloc[0]["Protein ID"]
         == "sp|P02769|ALBU_BOVIN Serum albumin OS=Bos taurus GN=ALB PE=1 SV=4"
     )
-    assert df.df.iloc[0]["Sequence Pre AA"] == "K"
-    assert df.df.iloc[0]["Sequence Post AA"] == "L"
+    assert df.iloc[0]["Sequence Pre AA"] == "K"
+    assert df.iloc[0]["Sequence Post AA"] == "L"
```

Comment:
same as above - please also assert identity of the resulting df - not just first line :)